assets/xml/objects/object_tab.xml

```diff
@@ -1,63 +1,63 @@
-        <Texture Name="object_tab_Tex_0063E8" OutName="tex_0063E8" Format="ci8" Width="8" Height="8" Offset="0x63E8" />
-        <Texture Name="object_tab_Tex_006428" OutName="tex_006428" Format="ci8" Width="32" Height="32" Offset="0x6428" />
-        <Texture Name="object_tab_Tex_006828" OutName="tex_006828" Format="ci8" Width="16" Height="16" Offset="0x6828" />
-        <Texture Name="object_tab_Tex_006928" OutName="tex_006928" Format="ci8" Width="32" Height="32" Offset="0x6928" />
-        <Texture Name="object_tab_Tex_006D28" OutName="tex_006D28" Format="ci8" Width="32" Height="32" Offset="0x6D28" />
-        <Texture Name="object_tab_Tex_007128" OutName="tex_007128" Format="ci8" Width="16" Height="16" Offset="0x7128" />
-        <Texture Name="object_tab_Tex_007228" OutName="tex_007228" Format="ci8" Width="16" Height="16" Offset="0x7228" />
-        <Texture Name="object_tab_Tex_007328" OutName="tex_007328" Format="i8" Width="32" Height="32" Offset="0x7328" />
-        <Texture Name="object_tab_Tex_007728" OutName="tex_007728" Format="rgba16" Width="8" Height="16" Offset="0x7728" />
-        <Texture Name="object_tab_Tex_007828" OutName="tex_007828" Format="ci8" Width="8" Height="8" Offset="0x7828" />
-        <Texture Name="object_tab_Tex_007868" OutName="tex_007868" Format="ci8" Width="8" Height="16" Offset="0x7868" />
-        <Texture Name="object_tab_Tex_0078E8" OutName="tex_0078E8" Format="i8" Width="16" Height="16" Offset="0x78E8" />
-        <Texture Name="object_tab_Tex_0079E8" OutName="tex_0079E8" Format="i8" Width="8" Height="8" Offset="0x79E8" />
-        <Texture Name="object_tab_Tex_007A28" OutName="tex_007A28" Format="rgba16" Width="16" Height="16" Offset="0x7A28" />
+        <Animation Name="gBartenIdleAnim" Offset="0x758" />
+        <DList Name="gBartenPelvisDL" Offset="0x3E10" />
+        <DList Name="gBartenLeftHandDL" Offset="0x3EF0" />
+        <DList Name="gBartenLeftBroomDL" Offset="0x4270" />
+        <DList Name="gBartenLeftLowerArmDL" Offset="0x4438" />
+        <DList Name="gBartenLeftUpperArmDL" Offset="0x46C8" />
+        <DList Name="gBartenRightHandDL" Offset="0x47F8" />
+        <DList Name="gBartenRightLowerArmDL" Offset="0x4B70" />
+        <DList Name="gBartenRightUpperArmDL" Offset="0x4E00" />
+        <DList Name="gBartenChestDL" Offset="0x4F30" />
+        <DList Name="gBartenHeadDL" Offset="0x5178" />
+        <DList Name="gBartenLeftThighDL" Offset="0x5668" />
+        <DList Name="gBartenLeftLegDL" Offset="0x5758" />
+        <DList Name="gBartenLeftFootDL" Offset="0x5840" />
+        <DList Name="gBartenRightThighDL" Offset="0x5988" />
+        <DList Name="gBartenRightLegDL" Offset="0x5A78" />
+        <DList Name="gBartenRightFootDL" Offset="0x5B60" />
+        <Texture Name="gBartenTLUT" OutName="barten_tlut" Format="rgba16" Width="16" Height="16" Offset="0x5CA8" />
+        <Texture Name="gBartenTlutTex" OutName="barten_tlut_texture" Format="rgba16" Width="16" Height="16" Offset="0x5EA8" />
```

Comment:
Is this also a TLUT or is this an actual texture? Could you elaborate what this is?

Reply:
the gBartenTLUT is the actual TLUT 

gBartenTlutTex seems to work as a TLUT but still as a texture format, it appears in object_tab.c
 "gsDPLoadTLUT_pal256(gBartenTlutTex)"